back/engines/free/email_campaigns/app/mailers/email_campaigns/event_registration_confirmation_mailer.rb

```diff
@@ -10,51 +11,76 @@ class EventRegistrationConfirmationMailer < ApplicationMailer
       :event_title, :project_title
 
     def campaign_mail
-      attachments['event.ics'] = Events::IcsGenerator.new.generate_ics(
-        Event.find(event.event_attributes.id),
-        locale.to_s
-      )
+      # TODO: Generate a preview ICS instead? Or move to the campaign / preview service?
```

Comment:
So if I understand correctly, the ICS file is an attachment in the email? I guess attachments are not included in the preview?

Reply:
Yes that's correct. I'm going to add a separate ticket for these TODOs